doc/intro.adoc

```diff
@@ -365,7 +365,7 @@ buffer to be `static`:
 
 [source]
 ----
-fn get_the_buffer() -> &'static [u8; 1024] { <1>
+fn get_the_buffer() -> &'mut static [u8; 1024] { <1>
```

Comment:
I didn't try compiling/running this or anything but I believe this is correct?

Reply:
You're right that the example is wrong, though the fix should be `&'static mut`. I'll update it and get this merged.

Reply:
Ha woops thanks